prijateli_tree/app/routers/games.py

```diff
@@ -420,16 +425,19 @@ def integrated_game(
         score_to_denirs(game_id, player_id, db, current_round)
 
 
-@router.post("/ready")
+@router.post("/player_ready")
 def confirm_player(
     player_id: int,
     game_id: int,
     db: Session = Depends(get_db),
+    body: dict = Body(...),
 ):
     """
     Confirms if the player is ready for the game
     """
 
+    player_id = body.get("player_id")
+    game_id = body.get("game_id")
```

Comment:
Within the rest of the platform, we use parameters for values. I'd like to stick with that.